src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/LaunchProfilesDebugLaunchProvider.cs

```diff
@@ -88,9 +88,9 @@ public override Task<IReadOnlyList<IDebugLaunchSettings>> QueryDebugTargetsAsync
     /// This is called on F5 to return the list of debug targets. What is returned depends on the debug provider extensions
     /// which understands how to launch the currently active profile type.
```

Comment:
This comment talks about the active profile, but with the new parameter it's no longer always correct.